Jenkinsfile

```
@@ -1054,7 +1050,6 @@ pipeline {
                                        inst_repos: el7_daos_repos,
                                        inst_rpms: 'daos-' + daos_packages_version +
                                                   ' daos-client-' + daos_packages_version +
-                                                  ' cart-' + env.CART_COMMIT + ' ' +
                                                   functional_rpms
```

```suggestion
                                                  ' ' + functional_rpms
```

This needs doing a number of times below.